tests/autoclassify/test_detect_intermittents.py

```diff
@@ -26,8 +26,7 @@ def test_detect_intermittents(test_job, failure_lines, classified_failures,
     MatcherManager._detector_funcs = {}
     detector = MatcherManager.register_detector(TestFailureDetector)
 
-    call_command('detect_intermittents', retriggered_job.repository.name,
-                 retriggered_job.guid)
+    call_command('detect_intermittents', str(test_job.id))
```

Comment:
I think I mentioned this on irc earlier, but I think the thing to do is move the code in the management command to a python module, then call it as part of the celery task. New relic suggests there's a large amount of overhead to the way we're currently doing things (since we need to start up a whole new python process, create a new database connection, etc). We could keep the management command as well, though honestly I doubt it would ever get used.